client/src/App.test.tsx

```diff
@@ -1,4 +1,3 @@
-import React from "react";
 import { render, screen } from "@testing-library/react";
 import App from "./App";
 
```

Comment:
Why is removing the react import necessary? If it's not, lets leave it out of this PR.